modules/migration-configuring-mcg.adoc

```diff
@@ -11,30 +11,38 @@
 = Retrieving Multicloud Object Gateway credentials
 
 ifdef::installing-3-4,installing-mtc[]
-You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint in order to configure MCG as a replication repository for the {mtc-full} ({mtc-short}).
+You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint to configure MCG as a replication repository for the {mtc-full} ({mtc-short})
+
+You must retrieve the Multicloud Object Gateway (MCG) credentials to create a `Secret` custom resource (CR) for {mtc-short}.
+endif::[]
+
+ifdef::installing-oadp-mcg[]
+You must retrieve the Multicloud Object Gateway (MCG) credentials to create a `Secret` custom resource (CR) for the OpenShift API for Data Protection (OADP).
 endif::[]
-You must retrieve the Multicloud Object Gateway (MCG) credentials in order to create a `Secret` custom resource (CR) for the OpenShift API for Data Protection (OADP).
 //ifdef::installing-oadp-mcg[]
 //endif::[]
 
-MCG is a component of {rh-storage}.
+[NOTE]
+====
+Although the MCG Operator was link:https://catalog.redhat.com/software/containers/ocs4/mcg-rhel8-operator/5ddbcefbdd19c71643b56ce9?architecture=amd64&image=64243f5dcd0eb61355af9abd[deprecated], the MCG plugin is still available for {rh-storage}. To download the plugin, browse to link:https://access.redhat.com/downloads/content/547/ver=4/rhel---9/4.15.4/x86_64/product-software[Download {rh-storage-first}] and download the appropriate MCG plugin for your operating system.
```

Comment:
_is deprecated_ is more common in our docs than _was deprecated_. The latter seem to be used with a specific version or event (_the feature was deprecated in version 7.23_)
```suggestion
Although the MCG Operator is link:https://catalog.redhat.com/software/containers/ocs4/mcg-rhel8-operator/5ddbcefbdd19c71643b56ce9?architecture=amd64&image=64243f5dcd0eb61355af9abd[deprecated], the MCG plugin is still available for {rh-storage}. To download the plugin, browse to link:https://access.redhat.com/downloads/content/547/ver=4/rhel---9/4.15.4/x86_64/product-software[Download {rh-storage-first}] and download the appropriate MCG plugin for your operating system.
```